src/helpers/sendEmail.ts

```diff
@@ -15,8 +15,13 @@ const emailer = createTransport({
   },
 })
 
-export default function (to: string, subject: string, secret: string) {
-  const { html } = generateTokenHtml({ secret })
+export default function (
```

Comment:
That's a lot args for one function, let's place them into the object

Reply:
![image](https://user-images.githubusercontent.com/49658988/188200852-15b10597-b303-4634-8882-89b4dbc06a95.png)

Reply:
Updated it. Thx